### ISSUE TITLE
WCAG contrast issues on search placeholder

### ISSUE THREAD

**Issue by owlnai** [makigas/makigas#711]
The current placeholder for the search box has a semi-opaque background. While it works 'fine' for text, the placeholder doesn't play really well and both WCAG 2.0 and WCAG 3.0 fail.

**Comment by danirod** [comment]
I'm going to keep this open, however, because now I am salty with Firefox and want to be reminded to do something with it.

**Comment by danirod** [comment]
After applying #713, I've measured of the foreground and background color of the placeholder text and I can see that it always passes WCAG AA for small text, so I am considering this task done.

:sparkles: However :sparkles: 

The measurements for Chrome (and I assume any other Blink-based browser like Edge) have been taken using 2x zoom or HDPI, since it seems that Chrome does a very weird job with the font smoothing and also does not support easily the properties for changing the font smoothing on operating systems that are not macOS. Subpixel antialiasing makes placeholder text appear lighter and thus it fails the WCAG tests, but this is a Chrome bug. #713 fixes the issue for other browsers.
